app/src/main/java/nu/yona/app/YonaApplication.java

```diff
@@ -202,5 +209,53 @@ public static Tracker getTracker()
 		}
 		return mContext.tracker;
 	}
+
+	protected OpenVPNService mService;
```

Comment:
Declare this member along with the others at the top of the class. Also strip the ``m`` prefix, as that's not a convention we apply.

Reply:
This one isn't addressed.